dbschema/src/main/java/org/folio/dbschema/Index.java

```diff
@@ -93,6 +93,7 @@ public void setSqlExpressionQuery(String sqlExpressionQuery) {
     this.sqlExpressionQuery = sqlExpressionQuery;
   }
 
+  @SuppressWarnings("java:S5852")//Line:105-Most of the regular expression engines use backtracking, but here regex is very small so we can keep it.
```

Comment:
The split regexp tries to do two tasks in one step:
1. Split on comma
2. Trim leading and tailing spaces.

The implementation `split(" *, *")` has a bug because it doesn't trim leading spaces of the first name and tailing spaces of the last name.

How about this code that avoids suppressing the regexp warning, fixes the bug, and is easier to read for me:

```
    String [] splitIndex = this.getMultiFieldNames().split(",");
    Arrays.asList(splitIndex).replaceAll(s -> s.strip());
```